contrib/typeshed/nacl/secret.pyi

```diff
@@ -0,0 +1,13 @@
+from nacl import encoding as encoding
+from nacl.utils import EncryptedMessage as EncryptedMessage, StringFixer as StringFixer, random as random
+from typing import Any, Optional
+
+class SecretBox(encoding.Encodable, StringFixer):
+    KEY_SIZE: Any = ...
+    NONCE_SIZE: Any = ...
+    MACBYTES: Any = ...
+    MESSAGEBYTES_MAX: Any = ...
+    def __init__(self, key: Any, encoder: Any = ...) -> None: ...
```

Comment:
```suggestion
    def __init__(self, key: SupportsBytes, encoder: Any = ...) -> None: ...
```

Reply:
SupportBytes also was producing an error for mypy but switching to bytes seems to be working.